products/models.py

```diff
@@ -55,8 +68,33 @@ class Product(AuditTimeStamp):
     description = models.CharField(
         _("Product Description"), max_length=1000, blank=True
     )
+
+    image = models.ImageField(upload_to=image_path, default="default_image.png")
     name = models.CharField(_("Product Name"), max_length=100)
 
+    def get_price(self):
+        """
+        Returns price with currency
+        Returns:
+            (str): Value containing price and currency
+        """
+        return f"Rs. {self.price}"
+
+    def save(self, *args, **kwargs):
+        # pylint: disable=no-member
+        if self.id is not None:
+            product = Product.objects.get(id=self.id)
+            if product.image.name != "default_image.png":
```

Comment:
Probably we can name the default as `product_image.png`. And this is something that we might need to add to the constants since it is used at multiple places.
```suggestion
            if product.image.name != "default_image.png":
```

Reply:
Done